src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -38,6 +42,10 @@ public static WebElement getElement(final WebElement element) {
      * @param element to click.
      */
     public static void clickElement(final WebElement element) {
+
+        if (THEME.equalsIgnoreCase("classic")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "classic" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)